lib/tlIO/FFmpegWrite.cpp

```diff
@@ -313,8 +313,11 @@ namespace tl
             switch (info.pixelType)
             {
             case image::PixelType::L_U8:
+            case image::PixelType::L_U16:
             case image::PixelType::RGB_U8:
             case image::PixelType::RGBA_U8:
+            case image::PixelType::RGB_U16:
```

Comment:
If you could group the RGB enums together, otherwise this looks fine. Like:
```
case image::PixelType::RGB_U8:
case image::PixelType::RGB_U16:
case image::PixelType::RGBA_U8:
case image::PixelType::RGBA_U16:
```